hack/ensure-go.sh

```diff
@@ -38,7 +38,7 @@ EOF
   local go_version
   IFS=" " read -ra go_version <<< "$(go version)"
   local minimum_go_version
-  minimum_go_version=go1.20.4
+  minimum_go_version=go1.20.0
```

Comment:
Just used to allow us to test using the krte image which currently uses go 1.20.3